### PR TITLE
Video about custom Suggester.

### PR DESCRIPTION
**Pull request by rodrigogiraoserrao**

### CODE REVIEW

File: input-with-custom-suggestions/myapp.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from pathlib import Path
+
+from textual.app import App, ComposeResult
+from textual.suggester import Suggester
+from textual.widgets import Input
+
+
+class FileSuggester(Suggester):
+    async def get_suggestion(self, value: str) -> str | None:
+        path = next(Path().glob(f"{value}*"), None)
```
Comment (willmcgugan) [inline input-with-custom-suggestions/myapp.py:12]: Did you intend to allow the full glob syntax here?

**Review by willmcgugan — APPROVED** [review]

File: input-with-custom-suggestions/myapp.py (hunk 1 above)
Comment (rodrigogiraoserrao) [reply input-with-custom-suggestions/myapp.py:12]: Yeah, why not?
I use the glob to find files whose names start with what I already typed and then `next` produces only the first match.

Comment (willmcgugan) [reply input-with-custom-suggestions/myapp.py:12]: The glob syntax can be absolute and include back references. In practice you would want to limit the file to a given directory.

Comment (rodrigogiraoserrao) [reply input-with-custom-suggestions/myapp.py:12]: Ah, ok, I get what you're getting at.
I don't think we want/need to worry about that for this short demo.

**Pull request merged**